src/Umbraco.Core/PropertyEditors/StringValueConverter.cs

```diff
@@ -0,0 +1,47 @@
+using Umbraco.Cms.Core.Models.PublishedContent;
+using Umbraco.Cms.Core.PropertyEditors.DeliveryApi;
+using Umbraco.Cms.Core.Templates;
+
+namespace Umbraco.Cms.Core.PropertyEditors;
+
+[DefaultPropertyValueConverter]
+public class StringValueConverter : PropertyValueConverterBase, IDeliveryApiPropertyValueConverter
+{
+    private static readonly string[] PropertyTypeAliases =
+    {
+        Constants.PropertyEditors.Aliases.String
+    };
+
+    public override bool IsConverter(IPublishedPropertyType propertyType)
+        => PropertyTypeAliases.Contains(propertyType.EditorAlias);
+
+    public override Type GetPropertyValueType(IPublishedPropertyType propertyType)
+        => typeof(string);
+
+    public override PropertyCacheLevel GetPropertyCacheLevel(IPublishedPropertyType propertyType)
+        => PropertyCacheLevel.Snapshot;
+
+    public override object? ConvertSourceToIntermediate(IPublishedElement owner, IPublishedPropertyType propertyType, object? source, bool preview)
+    {
+        if (source == null)
+        {
+            return null;
+        }
+
+        return source.ToString();
+    }
```

Comment:
```suggestion
    => source as string;
```

`source` can be anything here. The cautious (and less verbose) way is to use `as string` instead of `ToString()`. The latter will work for any object, and the outcome is in principle unknown.